keep-ui/app/settings/api-key-settings.tsx

```diff
@@ -19,29 +37,114 @@ interface Props {
 export default function ApiKeySettings({ accessToken, selectedTab }: Props) {
   const apiUrl = getApiURL();
   const { data, error, isLoading } = useSWR<ApiKeyResponse>(
-    selectedTab === "api-key" ? `${apiUrl}/settings/apikey` : null,
-    (url) => fetcher(url, accessToken),
+    selectedTab === "api-key" ? `${apiUrl}/settings/apikeys` : null,
+    async (url) => {
+        const response = await fetcher(url, accessToken);
+        setApiKeys(response.apiKeys);
+        return response;
+    },
     { revalidateOnFocus: false }
   );
 
+
+  const { data: configData } = useSWR<Config>("/api/config", fetcher, {
+    revalidateOnFocus: false,
+  });
+
+
+  const [isApiKeyModalOpen, setApiKeyModalOpen] = useState(false);
+  const [apiKeys, setApiKeys] = useState([]);
+
   if (isLoading) return <Loading />;
   if (error) return <div>{error.message}</div>;
 
-  const copyBlockApiKeyProps = {
-    theme: { ...a11yLight },
-    language: "text",
-    text: data?.apiKey || "",
-    codeBlock: true,
-    showLineNumbers: false,
-  };
+  const getCopyBlockProps = (secret: string) => {
+    return {
+        theme: { ...a11yLight },
+        language: "text",
+        text: secret,
+        codeBlock: true,
+        showLineNumbers: false,
+      }; 
+  }
+
+  // Determine runtime configuration
+  const authType = configData?.AUTH_TYPE as AuthenticationType;
+  // Create API key disabled if authType is none
+  const createApiKeyEnabled = authType !== AuthenticationType.NO_AUTH
 
   return (
     <div className="mt-10">
-      <Title>API Key</Title>
+      <div className="flex justify-between">
+        <div className="flex flex-col">
+          <Title>API Keys</Title>
+          <Subtitle>Add, edit or deactivate API keys from your tenant</Subtitle>
+        </div>
+
+        <div>
+          <Button
+            color="orange"
+            size="md"
+            icon={KeyIcon}
+            onClick={() => setApiKeyModalOpen(true)}
+            disabled={!createApiKeyEnabled}
+            tooltip={!createApiKeyEnabled ? "API Key creation is disabled because Keep is running in NO_AUTH mode.": "Add user"}
+          >
+            Create API key 
+          </Button>
+        </div>
+      </div>
       <Card className="mt-2.5">
+       {apiKeys.length ? (
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableHeaderCell>Name</TableHeaderCell>
+              <TableHeaderCell className="text-right">Key</TableHeaderCell>
```

Comment:
Align the text to the left, now its centered:
![CleanShot 2024-01-29 at 14 23 48](https://github.com/keephq/keep/assets/12069200/a27e1371-babd-4239-acbc-c8068acbed26)

Reply:
im a bit confused by this. I think you're talking about the Name header cell.

Will align to the left but all the other table headers are aligned to the right.

fix: 826f4f740f44b4d0025d8a3e73baadd07a1f696e